### PR TITLE
S2U-21 SAK-49651 T&Q: Add config key input - MASTER

### PR DESCRIPTION
**Pull request by stetsche**
https://sakaiproject.atlassian.net/browse/S2U-21
https://sakaiproject.atlassian.net/browse/SAK-49651

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties [hunk 1]
```diff
@@ -423,6 +423,9 @@ seb_allow_spell_checking = Gaitu ortografia berrikuspena
 seb_config_upload = Safe Exam Browser konfiguratzeko fitxategia
 seb_config_upload_success = Ongi igo da konfigurazioko fitxategia
 seb_config_upload_error = Ezin izan da konfigurazioko fitxategia igo. Ziurtatu .seb luzapena duela eta ez dagoela enkriptatuta.
+seb_config_key=Allowed config key
+seb_config_key_info=The config key can only be set, after the assessment is published. You will need to download the (altered) configuration \
+again and insert the exam keys based on that file.
```
Comment (stetsche) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:428]: Hi @PabloSanRoman! Could you help me with those new translations please?
The `seb_exam_keys_recommended` property also got updated, could you apply it as well?
image
```
seb_exam_keys_recommended=Azterketara sartzeko pasahitza jartzea gomendagarria da, Safe Exam Brower-en konfigurazioa agian ez baita automatikoki balidatzen.
```

Comment (PabloSanRoman) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:428]: Hi @stetsche  , here!: 

seb_exam_keys_recommended=Pasahitz bat ezartzea gomendagarria da, konfigurazioan edo azterketetan, Safe Exam Browser tresnaren konfigurazioa ezin delako automatikoki baliozkotu.

Comment (stetsche) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:428]: Thanks @PabloSanRoman! Could you also translate the new properties, please?
```
seb_config_key=Allowed config key
seb_config_key_info=The config key can only be set, after the assessment is published. You will need to download the (altered) configuration \
again and insert the exam keys based on that file.
```

Comment (PabloSanRoman) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:428]: Hi @stetsche , you are welcome!: 

seb_config_key=Baimendu konfigurazioko pasahitza
seb_config_key_info=Konfigurazioko pasahitza ezarri daiteke soilik azterketa publikatu ondoren. Aldatutako konfigurazioa 
beharko duzu deskargatu berriz. Gero sartu azterketaren pasahitza, fitxategi horrekin zerikusia duena.

Comment (stetsche) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages_eu.properties:428]: Nice, thanks @PabloSanRoman.

**Comment by jesusmmp** [comment]
@stetsche Where is the PR to s2u-22x? Thanks!

**Comment by stetsche** [comment]
I added the EU translations now and created the PR for s2u-22x.
@jesusmmp here it is: https://github.com/sakaiproject/sakai/pull/12295

**Comment by stetsche** [comment]
@mpellicer Could you re-run the maven build please. The webcomponents build failed when downloading node.

**Comment by jesusmmp** [comment]
> @mpellicer Could you re-run the maven build please. The webcomponents build failed when downloading node.

Done :)

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
This is cool, tested in #12295. Good job @stetsche

**Pull request merged**